ichspi.c

```diff
@@ -1289,6 +1289,13 @@ struct hwseq_data {
 	struct fd_region fd_regions[MAX_FD_REGIONS];
 };
 
+#define MAX_PR_REGISTERS 6
```

Comment:
does this come from a specification?

Reply:
No i used largest value that could be returned in https://github.com/Dasharo/flashrom/blob/ababbe13e6b06be551cc0f28a538b4747fcd2115/ichspi.c#L2108